canonical-webteam/deployment-configs
# Add www redirects to sites

Some of our sites lack a www redirect. Set them up much like Canonical and snapcraft

## QA

- Start up relevant sites with the `./qa-deploy` script and edit hosts file with the domain and the www. variant
The www. should redirect
